data_structures/src/data_request.rs

```diff
@@ -477,16 +505,37 @@ where
 
         // Collateral division rest goes for the miner
         let non_reveals_count = commits_count - reveals_count;
-        let (reward, _rest) = calculate_witness_reward(
-            commits_count,
-            reveals_count,
-            liars_count + non_reveals_count,
-            errors_count,
-            dr_output.witness_reward,
-            collateral,
-        );
-
-        let mut outputs: Vec<ValueTransferOutput> = if reveals_count > 0 {
+        let _after_second_hard_fork = true;
+        let (reward, _rest) = if _after_second_hard_fork {
```

Comment:
why do not you use the real after_second_hard_fork function?

Reply:
Because I need the block epoch